### PR TITLE
Allow custom host configuration

### LINKED ISSUE #3: Redirects to herokuapp
My instance of this service redirects me to https://badging.herokuapp.com, probably due to hardcoded url here: https://github.com/DNNX/badging/blob/master/config/prod.exs#L16

### PR DESCRIPTION
**Pull request by ertrzyiks**
It resolves #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ heroku create --buildpack "https://github.com/HashNuke/heroku-buildpack-elixir.g
 heroku config:set POOL_SIZE=18 \
                   READ_TOKEN=<randomly generated string> \
                   WRITE_TOKEN=<any other random string> \
+                  HOST=<your host name> \
```
Comment (DNNX) [inline README.md]: Please also change `export BADGING_URL=https://badging.herokuapp.com` to `export BADGING_URL=https://<your host name>`

**Review by DNNX — CHANGES_REQUESTED** [review]
Looks great, thanks! Just one little thing.

**Comment by ertrzyiks** [comment]
@DNNX readme updated, is it good now?

**Review by DNNX — APPROVED** [review]
Perfect!

**Pull request merged**